src/intermediate/SPECIFICATION.md

```
@@ -0,0 +1,460 @@
+  Counter1 = %Counter(12);
+  ret Counter1;
+}
+
+```
+
+Differently of default values, special values are primitives that are expected to exist on the runtime we are compiling to.
+
+### UI Operations
+Anything on the IR that initializes with '@' and is being used as an instruction, is an specific UI Operation, which determine what the UI itself should do. If being used as a value, then it's the visual reference to a handle of some internal string
+On Components, @binds are way to determine which value on the component should update which dependency. On the %Counter example above, we had
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)